_sass/footer.scss

```diff
@@ -1,2 +1,45 @@
 /* Footer Styles */
 
+footer {
+	max-height: 265px;
+	color: $navy;
+}
+
+.wrapper {
+	max-height: 190px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.footer_nav {
+	text-align: right;
+	letter-spacing: .1px;
+	margin: 41px 16px;
+}
+
+ul {
+	list-style-type: none;
```

Comment:
These styled global elements (ul, a, p) should be in a global scss file since they are not used only by the footer. If some of these elements require a unique footer styling then let's apply a more specific identifier, like a class name (e.g. footer_link).

Reply:
Moved this style to main.scss and I would agree that we should follow the convention of using a more specific identifier.